ceph-defaults/defaults/main.yml

```diff
@@ -0,0 +1,40 @@
+---
+# Preflight variables
+ceph_origin: community
+ceph_dev_branch: master
+ceph_dev_sha1: latest
+ceph_rhcs_version: 5
+ceph_mirror: https://download.ceph.com
+ceph_stable_key: https://download.ceph.com/keys/release.asc
+ceph_release: pacific
+upgrade_ceph_packages: false
+ceph_pkgs:
+  - lvm2
+  - chrony
+  - cephadm
```

Comment:
Does `yum install cephadm` install these others automatically? Can we reduce the duplication here?